### DISCUSSION TITLE
problemi con le automazioni

### DISCUSSION THREAD

**Discussion by Macinaie** [indomus/forum#3034] [HUB: Home Assistant]
Ciao a tutti, ho un problema con le automazioni non riesco più a visualizzarle, eppure dentro a dispositivi e servizi  entità sono ancora tutte presenti anche se in stato 'non disponibile'. Inoltre se cerco di crearne di nuove non riesco a salvarle.

**Comment by snakuzzo** [comment]
In base a quello che hai scritto nessuno potrà mai darti aiuto.
Devi dettagliare bene il problema, altrimenti la discussione può anche essere cancellata

**Reply by Macinaie** [reply]
cliccando su automazioni mi appare il seguente messaggio 'Le automazioni fanno sì che Home Assistant risponda automaticamente a ciò che accade all'interno e intorno alla casa.Translation Error: The intl string context variable "utente" was not provided to the string "Le automazioni collegano i trigger alle azioni in un modo ''quando trigger allora azione'' con condizioni opzionali. Ad esempio: "Quando il sole tramonta e se {utente} è in casa, accendi le luci"."' e non visualizzo più quelle che avevo creato. Se tento di crearne una nuova mi fà fare tutta la procedura ma alla fine quando vado a salvarla riappare il messaggio sopraindicato senza salvare nulla. Sono entrato dentro a dispositivi e servizi entità e le automazioni create sono ancora tutte presenti ma se clicco sopra mi appare il seguente messaggio 'Questa entità non viene più fornita dall'integrazione automation. Se l'entità non è più utilizzata, eliminala nelle impostazioni.' inoltre indica che non è disponibile. Nella colonna stato delle entità appare un cerchio con la freccia rosso e un punto esclamativo al centro e andando sopra con il puntatore appare la scritta ripristinato ma cliccandoci non accade nulla

**Reply by snakuzzo** [reply]
Non si capisce una fava di quello che hai scritto. Potresti metterlo in maniera un po' più ordinata ?
Grazie

**Comment by trebestie** [comment]
Che tipo di installazione hai?
